Makefile

```diff
@@ -54,6 +53,9 @@ fmt:
 gitlint:
 	gitlint --commits "origin..HEAD"
 
+prepare_version:
+	echo -e "package cmd\n\nconst Version = \"$$(git describe --tags)\"" > cmd/version.go
```

Comment:
Would be useful to `cat` the generated file here, to see that the contents are ok.
Especially good for CI logs.

Reply:
Looks to be done with commit 08951e38e856650fa6720bc18ab90956885b2eac